### PR TITLE
[Added Morse Code 🚀] :  Added the morse code

### PR DESCRIPTION
**Pull request by Blacksujit**
## added the Morse Code :

Added the Morse Code 

#hactobersfest2024

PR to issue no  --> #4

### CODE REVIEW

**Review by d-coder111 — CHANGES_REQUESTED** [review]
Add in the appropriate folder 📂. Kindly star this repo to show your support✨.

**Comment by Blacksujit** [comment]
Please check the PR i have created an seperate python folder and added the python codes in it  as you have said

**Review by d-coder111 — CHANGES_REQUESTED** [review]
See, the whole repo is all about python and out of the created folders, you have to choose to which particular folder(category) your program belongs to and add that into the folder. You don't have to create a new folder separately and if you have a bigger project then create a folder separately but choose the folder out of the created ones.
Kindly star the repository to show your support and encouragement. It’s designed to help programmers of all levels, from beginners to advanced, contribute and grow.

**Comment by Blacksujit** [comment]
Cool understand 👍

**Comment by Blacksujit** [comment]
can you review it once i have added the python codes in game galore and deleted the python codes folder

**Comment by Blacksujit** [comment]
if you have any problem regarding  so please give the feedback

**Review by Blacksujit — COMMENTED** [review]
all ok

**Review by d-coder111 — CHANGES_REQUESTED** [review]
Morse code is not a game, so try choosing an apt folder not the game-galore folder.

**Comment by Blacksujit** [comment]
Review, once and tell me

File: Game-Galore/AI_Guess_Animal.py
Comment (Blacksujit) [inline Game-Galore/AI_Guess_Animal.py:1]: This is cool

**Review by Blacksujit — COMMENTED** [review]
All ok

**Review by d-coder111 — CHANGES_REQUESTED** [review]
If your project is of 2 or more files..put them under a folder and add that folder inside game-galore folder. Also, star the repo to ensure your pull request gets merged!

**Comment by Blacksujit** [comment]
please , review

**Review by Blacksujit — COMMENTED** [review]
All ok

**Comment by d-coder111** [comment]
Kindly star this repo to ensure your pr gets merged.

**Comment by Blacksujit** [comment]
its alerady starred🚀🚀

**Review by d-coder111 — APPROVED** [review]
Looks ok👍..keep coding..keep contributing ✌️

**Pull request merged**